test/9.expect.js

```diff
@@ -0,0 +1,21 @@
+// TODO: Not sure if this actually is the wanted output.
```

Comment:
Note this TODO! Do you want me to blank this array out until the bug is fixed, or should we keep this for now?